### ISSUE TITLE
Unhandled crash when changing concurrent tasks between two conversions

### ISSUE THREAD

**Issue by Avasam** [milankarman/auto-loadless#18]
Steps before the crash:
- I set my concurrent tasks to 99
- Clicked convert and load
- Let the conversion do its thing
- Change the concurrent tasks to 12
- Clicked convert and load
- Start the conversion
- It's stuck at 0% for a few seconds before crashing the entire app (no crash handler)

Edit: It is easily replicatable. As long as the number of maximum concurrent tasks changes between two conversions.

This is from Windows' event viewer
```csharp
Application: Unload.exe
CoreCLR Version: 4.700.21.56803
.NET Core Version: 3.1.22
Description: The process was terminated due to an unhandled exception.
Exception Info: Xabe.FFmpeg.Exceptions.ConversionException: ffmpeg version 4.4-full_build-www.gyan.dev Copyright (c) 2000-2021 the FFmpeg developers
  built with gcc 10.2.0 (Rev6, Built by MSYS2 project)
  configuration: --enable-gpl --enable-version3 --enable-static --disable-w32threads --disable-autodetect --enable-fontconfig --enable-iconv --enable-gnutls --enable-libxml2 --enable-gmp --enable-lzma --enable-libsnappy --enable-zlib --enable-librist --enable-libsrt --enable-libssh --enable-libzmq --enable-avisynth --enable-libbluray --enable-libcaca --enable-sdl2 --enable-libdav1d --enable-libzvbi --enable-librav1e --enable-libsvtav1 --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxvid --enable-libaom --enable-libopenjpeg --enable-libvpx --enable-libass --enable-frei0r --enable-libfreetype --enable-libfribidi --enable-libvidstab --enable-libvmaf --enable-libzimg --enable-amf --enable-cuda-llvm --enable-cuvid --enable-ffnvcodec --enable-nvdec --enable-nvenc --enable-d3d11va --enable-dxva2 --enable-libmfx --enable-libglslang --enable-vulkan --enable-opencl --enable-libcdio --enable-libgme --enable-libmodplug --enable-libopenmpt --enable-libopencore-amrwb --enable-libmp3lame --enable-libshine --enable-libtheora --enable-libtwolame --enable-libvo-amrwbenc --enable-libilbc --enable-libgsm --enable-libopencore-amrnb --enable-libopus --enable-libspeex --enable-libvorbis --enable-ladspa --enable-libbs2b --enable-libflite --enable-libmysofa --enable-librubberband --enable-libsoxr --enable-chromaprint
  libavutil      56. 70.100 / 56. 70.100
  libavcodec     58.134.100 / 58.134.100
  libavformat    58. 76.100 / 58. 76.100
  libavdevice    58. 13.100 / 58. 13.100
  libavfilter     7.110.100 /  7.110.100
  libswscale      5.  9.100 /  5.  9.100
  libswresample   3.  9.100 /  3.  9.100
  libpostproc    55.  9.100 / 55.  9.100
Input #0, mov,mp4,m4a,3gp,3g2,mj2, from 'C:\Users\Avasam\Desktop\[WR] Pitfall_ The Lost Expedition Any% 24_32.mp4':
  Metadata:
    major_brand     : mp42
    minor_version   : 0
    compatible_brands: isommp42
    creation_time   : 2020-10-10T07:15:25.000000Z
  Duration: 00:25:01.45, start: 0.000000, bitrate: 1500 kb/s
  Stream #0:0(und): Video: h264 (High) (avc1 / 0x31637661), yuv420p(tv, bt709), 1280x720 [SAR 1:1 DAR 16:9], 1369 kb/s, 30 fps, 30 tbr, 15360 tbn, 60 tbc (default)
    Metadata:
      creation_time   : 2020-10-10T07:15:25.000000Z
      handler_name    : ISO Media file produced by Google Inc. Created on: 10/10/2020.
      vendor_id       : [0][0][0][0]
  Stream #0:1(und): Audio: aac (LC) (mp4a / 0x6134706D), 44100 Hz, stereo, fltp, 128 kb/s (default)
    Metadata:
      creation_time   : 2020-10-10T07:15:25.000000Z
      handler_name    : ISO Media file produced by Google Inc. Created on: 10/10/2020.
      vendor_id       : [0][0][0][0]
Stream mapping:
  Stream #0:0 -> #0:0 (h264 (native) -> mjpeg (native))
Press [q] to stop, [?] for help
[swscaler @ 00000157d085dc40] deprecated pixel format used, make sure you did set range correctly
Output #0, image2, to 'C:\Users\Avasam\Desktop\WRPitfallTheLostExpeditionAny2432mp4_frames\%d.jpg':
  Metadata:
    major_brand     : mp42
    minor_version   : 0
    compatible_brands: isommp42
    encoder         : Lavf58.76.100
  Stream #0:0(und): Video: mjpeg, yuvj420p(pc, bt709, progressive), 640x360 [SAR 1:1 DAR 16:9], q=2-31, 200 kb/s, 30 fps, 30 tbn (default)
    Metadata:
      creation_time   : 2020-10-10T07:15:25.000000Z
      handler_name    : ISO Media file produced by Google Inc. Created on: 10/10/2020.
      vendor_id       : [0][0][0][0]
      encoder         : Lavc58.134.100 mjpeg
    Side data:
      cpb: bitrate max/min/avg: 0/0/200000 buffer size: 0 vbv_delay: N/A
frame=    1 fps=0.0 q=0.0 size=N/A time=00:00:00.00 bitrate=N/A speed=   0x    
[image2 @ 00000157ce320540] Could not open file : C:\Users\Avasam\Desktop\WRPitfallTheLostExpeditionAny2432mp4_frames\1.jpg
av_interleaved_write_frame(): I/O error
frame=    2 fps=0.0 q=3.0 Lsize=N/A time=00:00:00.03 bitrate=N/A speed=1.17x    
video:7kB audio:0kB subtitle:0kB other streams:0kB global headers:0kB muxing overhead: unknown
Conversion failed!
   at Xabe.FFmpeg.FFmpegWrapper.<>c__DisplayClass11_0.<RunProcess>b__0()
   at System.Threading.Tasks.Task`1.InnerInvoke()
   at System.Threading.ExecutionContext.RunInternal(ExecutionContext executionContext, ContextCallback callback, Object state)
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.Tasks.Task.ExecuteWithThreadLocal(Task& currentTaskSlot, Thread threadPoolThread)
--- End of stack trace from previous location where exception was thrown ---
   at Xabe.FFmpeg.Conversion.Start(String parameters, CancellationToken cancellationToken)
   at unload.VideoProcessor.ConvertToImageSequence(String inputPath, String outputPath, TimeSpan startTime, TimeSpan endTime, Int32 frameWidth, Int32 frameHeight, Double fps, CancellationTokenSource cts, Action`1 onProgress) in C:\Users\milan\repos\UnLoad\Unload\src\processors\VideoProcessor.cs:line 46
   at unload.ConvertWindow.<>c__DisplayClass7_0.<<btnConvert_Click>b__0>d.MoveNext() in C:\Users\milan\repos\UnLoad\Unload\src\windows\ConvertWindow.xaml.cs:line 116
--- End of stack trace from previous location where exception was thrown ---
   at System.Threading.Tasks.Task.<>c.<ThrowAsync>b__139_1(Object state)
   at System.Threading.QueueUserWorkItemCallback.<>c.<.cctor>b__6_0(QueueUserWorkItemCallback quwi)
   at System.Threading.ExecutionContext.RunForThreadPoolUnsafe[TState](ExecutionContext executionContext, Action`1 callback, TState& state)
   at System.Threading.QueueUserWorkItemCallback.Execute()
   at System.Threading.ThreadPoolWorkQueue.Dispatch()

```

**Comment by milankarman** [comment]
Going to leave this in the backlog for a bit as I hope it'll be resolved as a side effect of #19

**Comment by milankarman** [comment]
This is no longer an issue because the concurrent task setting was removed in version [1.3.0](https://github.com/milankarman/Unload/releases/tag/1.3.0)
